src/index.js

```diff
@@ -42,7 +42,7 @@ import Item from 'src/views/Items/Item';
 import Items from 'src/views/Items/Items';
 import Statistic from 'src/views/Statistic/Statistic';
 
-export default {
+const stardust = {
```

Comment:
Seems like a few of the changes in this PR relate back to not being able to transpile the Karma, config, which I believe calling the already available `babel-node` CLI command (made available by `babel-core` should resolve.